### PR TITLE
NFC TNEP OOB arrangements

### PR DESCRIPTION
**Pull request by KAGA164**
- Allows creation of the static or global OOB records
- Change order in the TNEP Initial Message
- Remove trash from sample testing

### CODE REVIEW

**Comment by KAGA164** [comment]
@grochu @kapi-no Please review

File: samples/nfc/tnep_tag/src/main.c [hunk 1]
```diff
@@ -14,10 +14,6 @@
 #include <nfc/tnep/tag.h>
 #include <nfc/ndef/msg.h>
 #include <nfc/ndef/text_rec.h>
-#include <logging/log.h>
-
-
-LOG_MODULE_REGISTER(dupa);
```
Comment (kapi-no) [inline samples/nfc/tnep_tag/src/main.c:20]: Farewell, funny line!

Comment (pdunaj) [reply samples/nfc/tnep_tag/src/main.c:20]: so sad :(

Comment (pdunaj) [reply samples/nfc/tnep_tag/src/main.c:20]: device under permanent association... what's wrong with that!?

Comment (grochu) [reply samples/nfc/tnep_tag/src/main.c:20]: :(

**Pull request merged**